src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -20,9 +20,40 @@ const BlogPost = ({ data }) => {
   const authorBlog = AuthorSerializer.deSerialize(author);
   const { setState } = useContext(AppContext);
   const [ disappearSocial, setDisappearSocial ] = useState(false);
-
   const categoryBlog = useCategory(data.mdx.frontmatter.category);
 
+
+  const checkWindow = () => {
+    if (typeof window !== 'undefined') {
```

Comment:
Just to know: what is the difference between your solution and using `if (window)`?

Reply:
No difference, only I wrote in that format.

Reply:
Done

Reply:
ok, so simplify this because the window can be null, and in the if(window) you can detect this case